pkgs/by-name/xf/xfsprogs/package.nix

```diff
@@ -79,6 +82,10 @@ stdenv.mkDerivation rec {
   # FIXME: forbidden rpath
   postInstall = ''
     find . -type d -name .libs | xargs rm -rf
+
+    substituteInPlace $bin/sbin/{xfs_scrub_all,xfs_protofile} \
+      --replace-fail '#!/usr/bin/python3' '#!/usr/bin/env python3'
+    wrapPythonProgramsIn "$bin/sbin" "$out $pythonPath"
```

Comment:
Why not put `python3.withPackages (ps: [ ps.dbus-python ])` in `buildInputs` instead?